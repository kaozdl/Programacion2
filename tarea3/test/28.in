# finitario
finitario (1,m) 1 .
finitario (21,a) 1 .
finitario (22,g) 2 .
finitario (321,b) 2 1 .
Fin
