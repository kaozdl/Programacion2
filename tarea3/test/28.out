1>
2>
0
(1,m)
3>(1,m)
(1,m)
1
(21,a)
4>(1,m)
(1,m)
(21,a)
2
(21,a)(22,g)
5>(1,m)
(1,m)
(21,a)(22,g)
2
(321,b)
6>Fin.
